frontend/src/views/Comment.svelte

```diff
@@ -0,0 +1,86 @@
+<style>
+  .comment {
+    white-space: pre;
+  }
+
+  button {
+    border: 0px;
+    filter: invert(20%);
+  }
+
+  button:hover {
+    filter: invert(40%);
+  }
+</style>
+
+<script>
+  import { tick } from "svelte";
+  import { currentPosition } from "../hex/stores.js";
+  import { selected, resourceNodeDataMap } from "../stores.js";
+  import Hoverable from "../utils/Hoverable.svelte";
+  import Icon from "../utils/Icon.svelte";
+  export let comment, rootResource, selfId;
+  let text = comment[1];
+  let addresses = text.matchAll("#[a-fA-F0-9]+[@0x[0-9a-fA-F]+]*", text);
```

Comment:
I tried doing just a hex address offset as the text, but that didn't get recognized and turned into a link/button. Is it supposed to?

Reply:
I felt it was redundant with the "range" we already have in the comments, but as i think about it it makes sense to be able to cross reference with other areas of the same resource without tagging its own resource_id